android/src/main/java/com/ryanheise/just_waveform/JustWaveformPlugin.java

```diff
@@ -19,52 +19,56 @@
 
     @Override
     public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBinding) {
-        channel = new MethodChannel(flutterPluginBinding.getFlutterEngine().getDartExecutor(), "com.ryanheise.just_waveform");
+        channel = new MethodChannel(flutterPluginBinding.getFlutterEngine().getDartExecutor(),
+                "com.ryanheise.just_waveform");
         channel.setMethodCallHandler(this);
     }
 
     @Override
     public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result) {
         switch (call.method) {
-        case "extract":
-            String audioInPath = call.argument("audioInPath");
-            String waveOutPath = call.argument("waveOutPath");
-            Integer samplesPerPixel = call.argument("samplesPerPixel");
-            Integer pixelsPerSecond = call.argument("pixelsPerSecond");
-            WaveformExtractor waveformExtractor = new WaveformExtractor(audioInPath, waveOutPath, samplesPerPixel, pixelsPerSecond);
-            waveformExtractor.start(new WaveformExtractor.OnProgressListener() {
-                @Override
-                public void onProgress(int progress) {
-                     Map<String, Object> args = new HashMap();
-                    args.put("progress", progress);
-                    args.put("waveOutFile",waveOutPath);
+            case "extract":
```

Comment:
Can you undo the change to this line and similar lines? Changes to the coding standards, such as whether cases should be indented or not, should be first discussed in an issue and then changed in its own pull request.

Reply:
Just to clarify the current coding style, each switch case is considered as a label, not an executable line of code, and you don't indent labels. Of course there are other coding styles, but this is just to clarify what is currently being used in the Java code.

Reply:
FYI, the original Java coding conventions from SUN Microsystems: https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#468